python/cudf/cudf_pandas_tests/third_party_integration_tests/tests/test_catboost.py

```diff
@@ -112,17 +125,17 @@ def test_catboost_with_categorical_features():
     "X, y",
     [
         (
-            pd.DataFrame(rng.standard_normal((100, 5))),
-            pd.Series(rng.standard_normal(100)),
+            pd.DataFrame(rng.standard_normal((1000, 5))),
+            pd.Series(rng.standard_normal(1000)),
         ),
-        (rng.standard_normal((100, 5)), rng.standard_normal(100)),
+        (rng.standard_normal((1000, 5)), rng.standard_normal(1000)),
     ],
 )
 def test_catboost_train_test_split(X, y):
     from sklearn.model_selection import train_test_split
 
     X_train, X_test, y_train, y_test = train_test_split(X, y, random_state=42)
-    model = CatBoostRegressor(iterations=10, verbose=0)
+    model = CatBoostRegressor(iterations=50, verbose=0)
```

Comment:
```suggestion
    model = CatBoostRegressor(iterations=50, random_state=708, verbose=0)
```

For all of these tests, you should pass in a non-0 seed to control the randomness internal to `catboost`. I'm not certain if `catboost` training is deterministic given the parameters you've supplied, but this is still a good idea just to be safe and reduce the risk of flakiness in these tests.